hack/jenkins/gotest-dockerized.sh

```diff
@@ -40,8 +37,9 @@ docker run --rm=true \
   -v /var/run/docker.sock:/var/run/docker.sock \
   -v "$(which docker)":/bin/docker \
   -v "${REPO_DIR}":/go/src/k8s.io/kubernetes \
-  -v "${KUBE_JUNIT_REPORT_DIR}":/workspace/artifacts \
+  -v "${WORKSPACE}/_artifacts":/workspace/artifacts \
   -v /etc/localtime:/etc/localtime:ro \
-  --env REPO_DIR="${REPO_DIR}" \
+  -e "KUBE_VERIFY_GIT_BRANCH=${KUBE_VERIFY_GIT_BRANCH:-}" \
```

Comment:
I forgot to add `KUBE_FORCE_VERIFY_CHECK` here, so I don't think it's going to run on post-commit Jenkins yet.